.travis.yml

```diff
@@ -0,0 +1,30 @@
+sudo: false
+
+language: php
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+    - vendor
+
+env:
+  global:
+    - COMPOSER_ARGS="--no-interaction"
+
+matrix:
+  include:
+    - php: 7
```

Comment:
Maybe should also test it on 7.1? That way we'll know if there's breakage if/when we want to use it against that version instead of 7.0.